tide_site.module

```diff
@@ -44,6 +45,9 @@ function tide_site_entity_bundle_create($entity_type_id, $bundle) {
         // Create/update form display. View display is not created to avoid
         // unexpected fields appearing in UI.
         $field_config = $fields_helper->provisionField($field_name, $entity_type_id, $bundle);
+        $field = FieldConfig::loadByName($entity_type_id, $bundle, $field_config['field_name']);
```

Comment:
@vincent-gao Good catch.

The handling of the required property occurs [here](https://github.com/dpc-sdp/tide_site/blob/9c8b524a9f8cf61b0449286b70b92e3fc6426c23/src/TideSiteFields.php#L105)

This change should be made there.

Reply:
thanks @GROwen , changed.